docs/specifications/tests/DNSQueryAndResponseDefaults.md

```diff
@@ -0,0 +1,165 @@
+|:---------------------|:--------------------|:-----|:-------|
+|EDNS                  | OPT record included | X    |        |
+|EDNS UDP Message size | 512                 |      |        |
+|EDNS Extended RCODE   | no                  |      |        |
+|EDNS Version          | 0                   |      |        |
+|EDNS DO flag          | unset               |      |        |
+|EDNS Z flag           | unset               |      |        |
+|EDNS0 Option          | none                |      |        |
+
+
+## Default setting in *DNSSEC Query*
+
+A *DNSSEC Query* inherits the default setting from an *EDNS Query* except for the
+parameter specified below. If a *Parameter* is specified as "fixed" (with an
+"X" in that column) then the default value cannot be overidden.
+
+A test case specification can refer to a *DNSSEC Query* with one or several
+changes to the Parameters.
+
+|Parameter             |Default value |Fixed |Comment |
+|:---------------------|:-------------|:-----|:-------|
+|EDNS DO flag          | set          | X    |        |
+|EDNS UDP Message size | 1280         |      |        |
+
+
+## Default handling of a *DNS Response*
+
+A *DNS Response* is a response to a *DNS Query*. Unless specified in the test
+case specification, the items in the response are handled as listed. If a
+*Response Item* is specified as "fixed" (with an "X" in that column) then the
+requirement, as specified under "Default handling", must be met for the response
+to be considered to be a DNS response.
+
+|Response Item |Default handling                          | Fixed | Comment              |
+|:-------------|:-----------------------------------------|:------|:---------------------|
+|OpCode        | Require value to be "response"           | X     |                      |
+|QR flag       | Require flag to be set                   | X     |                      |
+|AA flag       | -                                        |       | Defined in test case |
+|TC flag       | Re-query over TCP if set                 |       |                      |
+|RD flag       | ignore                                   |       |                      |
+|RA flag       | ignore                                   |       |                      |
+|AD flag       | ignore                                   |       |                      |
+|CD flag       | ignore                                   |       |                      |
+|RCODE         | -                                        |       | Defined in test case |
+|Query name    | ignore                                   |       |                      |
+|Query type    | ignore                                   |       |                      |
+|Query class   | Require value to be same as in the query |       | Normally "IN"        |
+|EDNS          | ignore                                   |       |                      |
+
+* Check against query name and query type is, by default, done against the values
```

Comment:
For better readability, maybe change the "response item" values (i.e., query name, query type, etc) in the text below to be in italic for example? Or double quoted.

Reply:
Updated.

Reply:
Missing italic for "query name" and "query type" on line 112.

Reply:
When I read this again, I feel I have to check the wording of this sentence.

Reply:
Updated wording.